### DISCUSSION TITLE
Timeline

### DISCUSSION THREAD

**Discussion by michaelcattell** [dwp/design-system#446] [Q&A]
Hello, we'd like to use this open discussion on the use of the [**Timeline**](https://design-system.dwp.gov.uk/components/timeline):

**Some discussion points to get things going:**
Have you used or know where this component has been used?
Do you know if there's been any research or testing done on it or around it?
Do you agree with the user need?
Things we don’t know enough?

(Useful link: dwp/design-system-community-backlog#24 )

**Comment by jwContentDesign** [comment]
Starting to see some evidence in research that the grey font may be making content hard to scan. There's several DWP services working with the timeline component, so we can add more detail here as we get it. I'm working on an internal service and we will be testing layouts and fonts to help improve this, for example, the [HMRC timeline component](https://design.tax.service.gov.uk/hmrc-design-patterns/timeline/) uses spacing and black font.

**Reply by martinwake** [reply]
Thanks Jane - can you share any more detail? Do we know that it's definitely the colour making it hard to scan, rather than (for example) the size, weight or position of the text?

**Reply by jwContentDesign** [reply]
It could be a combination of colour and spacing, the font size is the standard size. We are going to test some variations of spacing, order and font colour, so will share findings when we have them.

**Reply by jwContentDesign** [reply]
[HMRC's issue comments on their timeline pattern](https://github.com/hmrc/design-patterns/issues/167#issue-729959741) explains the reason they changed hint text to standard body text was because users were also not spotting some content in testing.

**Comment by htmlandbacon** [comment]
Have we got any samples of a more realistic timeline?

I know an Access to work one is shown on here https://github.com/dwp/design-system-community-backlog/issues/24

**Comment by maxmarulli** [comment]
ATW have been using this pattern for both the Payments and Assess journeys and it has been testing in the first instance well with users for internal service. @rachelderbyshire may have some more information on the assess side of the journey. I'll look and see if any of the findings from UR would be useful to feed into this

**Reply by rosie-brave** [reply]
@maxmarulli @rachelderbyshire I'd be interested to learn more about your usage of the timeline for internal service users. We're considering it for an application's activity history for caseworkers. Any insight appreciated :)

**Reply by rosie-brave** [reply]
I'm particularly interested in how it performs with assistive text

**Comment by rosie-brave** [comment]
We've started to use this pattern to record actions taken when processing an application in an internal caseworking system. We're on our first round of testing.  So far, participants (all caseworkers) have indicated they'd prefer to see it in the opposite order - starting with the oldest at the top and the most recent at the bottom. This preference is informed by two things: 1) the way history is recorded in our legacy system is oldest first and 2) users need the full picture, not just the most recent update in order to do their work - therefore it feels logical to them to have oldest to newest. 
Just wondering if anyone else has encountered this?
